addons/file-sync/Commands/Integrity.ts

```diff
@@ -107,7 +107,7 @@ export default class Import extends BaseCommand {
 
   private async checkDatabaseIntegrity() {
     let skippedDeletions = 0;
-    for await (const file of await this.deps.File.find({})) {
+    for (const file of await this.deps.File.all()) {
```

Comment:
```suggestion
    for await (const file of this.deps.File.query()) {
```

We don't want to have all records in memory at once